h2o-r/h2o-package/R/frame.R

```diff
@@ -2439,8 +2424,7 @@ h2o.print <- function(x,n=6L) {
 #' @param cols Print the per-column str for the H2OFrame
 #' @export
 h2o.str <- function(object, ..., cols=FALSE) {
-  if(is.H2OFrame(object)) str(object, ..., cols)
-  else str(object, ..., cols)
+  str(object, ..., cols)
```

Comment:
AFAIK argument `cols` will be ignored, arguments passed after `...` should always be named.
```r
g = function(x, ..., cols=TRUE) {
    print(cols)
    return(invisible(x))
}
f = function(x, ..., cols=FALSE) {
    g(x, ..., cols)
}
f(1)
#[1] TRUE
f(1, cols=FALSE)
#[1] TRUE
```
it was a bug before :)